src/powershell/FinOpsToolkit.psm1

```diff
@@ -117,23 +117,23 @@ function Save-FinOpsHubTemplate
 
 <#
     .SYNOPSIS
-        Retrieves available version numbers of the FinOps toolkit.
```

Comment:
What's the standard here? I've seen spaces added and now spaces removed. What's the official guidance?

Reply:
No space is the correct way. Get-Help doesn't parse the keywords correctly if there are spaces. Which was news to me